pages/index.js

```diff
@@ -1,41 +1,60 @@
-import Head from 'next/head';
```

Comment:
index.js is now our "landing page" and is the first thing users will see upon accessing the site